server/server.js

```diff
@@ -2,13 +2,15 @@ import express from 'express';
 import cors from 'cors';
 import mongoose from 'mongoose';
 import dotenv from 'dotenv';
+import accountRoutes from './routes/accounts.js';
 
 dotenv.config();
 
 const app = express();
 
 app.use(express.json());
 app.use(cors());
+app.use('/api/accounts', accountRoutes);
```

Comment:
The routes don't work on my end if they have this `/api/` bit in, did you need to do anything to get that set up?